### PR TITLE
Spanish Translation of  guides/concepts/views.md

### PR DESCRIPTION
**Pull request by d4g0**

### CODE REVIEW

**Comment by debs-obrien** [comment]
Ready for review if anyone has a second @Kronhyx @alvarosaburido @Dawntraoz

File: views.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+---
+title: Vistas 
+description: La sección de vistas describe todo lo que necesitas saber para configurar los datos y las vistas para una ruta específica en tu aplicación Nuxt.js . Las Vistas consisten en una plantilla de la aplicación, un diseño (layout), y la propia página.
+position: 1
+category: concepts
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/02_concepts/01_views?fontsize=14&hidenavigation=1&theme=dark
+questions:
+- question: Cual es el orden de composición de una vista Nuxt (de arriba a abajo) ?
```
Comment (Kronhyx) [inline views.md:8]:
```suggestion
- question: Cuál es el orden de composición de una vista Nuxt (de arriba a abajo) ?
```

File: views.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+answers:
+- Layout → Page → Document
+- Page → Layout → Document
+- Document → Layout → Page
+correctAnswer: Document → Layout → Page
+- question: Como es llamado el diseño (layout) por defecto ?
```
Comment (Kronhyx) [inline views.md:14]:
```suggestion
- question: Cómo es llamado el diseño (layout) por defecto ?
```

File: views.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+answers:
+- default.vue
+- layout.vue
+- defaultLayout.vue
+correctAnswer: default.vue
+- question: Como crear un diseño(layout) personalizado ?
```
Comment (Kronhyx) [inline views.md:20]:
```suggestion
- question: Cómo crear un diseño(layout) personalizado ?
```

File: views.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+answers:
+- añadir un .vue archivo a la carpeta de pages 
+- añadir un .vue archivo a la carpeta de layouts 
+- añadir un .vue archivo a la carpeta de components 
+correctAnswer: añadir un .vue archivo a la carpeta de layouts 
+- question: Como activas tu diseño personalizado llamado 'blog' en tu página ?
```
Comment (Kronhyx) [inline views.md:26]:
```suggestion
- question: Cómo activas tu diseño personalizado llamado 'blog' en tu página ?
```

File: views.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+answers:
+- "layout: 'blog'"
+- "layout: 'default'"
+- "blog: 'blog'"
+correctAnswer: "layout: 'blog'"
+- question: Donde pones tu archivo error.vue para crear una página de error personalizada ?
```
Comment (Kronhyx) [inline views.md:32]:
```suggestion
- question: Dónde pones tu archivo error.vue para crear una página de error personalizada ?
```

File: views.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+La sección de vistas describe todo lo que necesitas saber para configurar los datos y las vistas para una ruta específica en tu aplicación Nuxt.js . Las Vistas consisten en una plantilla de la aplicación, un diseño (layout), y la propia página. Adicionalmente, se pueden definir etiquetas meta personalizadas para la sección de cabecera(head) de cada página, las cuales son importantes para el SEO (Search Engine Optimization).
+
+![Composition of a View in Nuxt.js](/guides/views.png)
+
+Composición de una vista en Nuxt.js
+
+## Páginas
+
+Cada página es un componente Vue, pero Nuxt.js añade algunos atributos y funciones adicionales para hacer el desarrollo de tu aplicación lo mas fácil posible.
```
Comment (Kronhyx) [inline views.md:48]:
```suggestion
Cada página es un componente Vue, pero Nuxt.js añade algunos atributos y funciones adicionales para hacer el desarrollo de tu aplicación lo más fácil posible.
```

File: views.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+
+```html{}[layouts/default.vue]
+<template>
+<Nuxt />
+</template>
+```
+
+<base-alert type="next">
+
+Aprende mas sobre [Nuxt component](/guides/features/nuxt-components) en el capítulo componentes.
```
Comment (Kronhyx) [inline views.md:98]:
```suggestion
Aprende más sobre [Nuxt component](/guides/features/nuxt-components) en el capítulo componentes.
```

**Review by Kronhyx — CHANGES_REQUESTED** [review]
Just misspelling

**Comment by debs-obrien** [comment]
fantastic. Thanks

**Pull request merged**